package.json

```diff
@@ -184,8 +184,7 @@
     },
     "patchedDependencies": {
       "babel-plugin-typescript-to-proptypes@1.4.2": "patches/babel-plugin-typescript-to-proptypes@1.4.2.patch",
-      "vite-bundle-analyzer@0.12.1": "patches/vite-bundle-analyzer@0.12.1.patch",
-      "react-modal@3.16.1": "patches/react-modal@3.16.1.patch"
```

Comment:
This patch was added on the `preview/fec-156-react19` branch but we won't be needing it anymore